tests/installation/release_notes_from_rpm.pm

```diff
@@ -1,36 +1,31 @@
 # SUSE's openQA tests
 #
-# Copyright © 2017-2018 SUSE LLC
+# Copyright © 2021 SUSE LLC
 #
 # Copying and distribution of this file, with or without modification,
 # are permitted in any medium without royalty provided the copyright
 # notice and this notice are preserved.  This file is offered as-is,
 # without any warranty.
 
 # Summary: Check origin of displayed releasenotes during installation
-# Maintainer: mgriessmeier <mgriessmeier@suse.com>, Nick Singer <nsinger@suse.de>
-# Tags: fate#323273, poo#26786
+# Maintainer: QA SLE YaST <qa-sle-yast@suse.de>
 
 use base 'y2_installbase';
 use strict;
 use warnings;
 use testapi;
-use version_utils ':VERSION';
 
 sub run {
     assert_screen('release-notes-button');
     send_key('ctrl-shift-alt-x');
     assert_screen('yast-xterm');
-    my $src = check_var('SCC_REGISTER', 'installation') ? "RPM" : "URL";
+    # This is releasenotes_offline so we expect release notes from RPM
```

Comment:
I would recommend to remove this comment, as you see, after a refactor of the code is even less meaningful.